spyder/plugins/explorer/tests/test_explorer.py

```diff
@@ -46,5 +66,99 @@ def test_project_explorer(project_explorer):
     assert project_explorer
 
 
+project_dir = osp.join(
+        getcwd_or_home(),
```

Comment:
Hanging indent should be 4 spaces

Reply:
It is modified now. it was a line continuation suggested by Spyder.

Reply:
Right; for whatever legacy reason Spyder inserts 8 spaces for hanging indents rather than the standard 4, and I've never gotten around to fixing it but it should be very simple to resolve.